heron/stmgr/src/cpp/manager/instance-server.cpp

```diff
@@ -463,6 +483,13 @@ void InstanceServer::StopBackPressureConnectionCb(Connection* _connection) {
   sp_string instance_name = GetInstanceName(_connection);
   CHECK_NE(instance_name, "");
 
+  LOG(INFO) << "We don't observe back pressure now on sending data to instance " << instance_name;
+
+  if (droptuples_upon_backpressure_) {
```

Comment:
Suggest to add a log here like "droptuples_upon_backpressure is set to true, hence no backpressure mechanism is initialized"

Reply:
Added